apiv2/database/migrations/versions/20241002_152402_autogenerated.py

```diff
@@ -115,13 +115,16 @@ def upgrade() -> None:
     op.add_column("tomogram", sa.Column("deposition_date", sa.DateTime(timezone=True), nullable=True))
     op.add_column("tomogram", sa.Column("release_date", sa.DateTime(timezone=True), nullable=True))
     op.add_column("tomogram", sa.Column("last_modified_date", sa.DateTime(timezone=True), nullable=True))
-    op.drop_column("tomogram", "is_canonical")
+
+    conn = op.get_bind()
+    backfill_sql = sa.sql.text("UPDATE tomogram SET is_portal_standard = is_canonical")
+    conn.execute(backfill_sql)
```

Comment:
I'm modifying an *existing* migration because this migration hasn't been executed in prod yet. I put the `is_canonical` column back in place manually in staging.